tests/linked_list_tests.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+from linked_list.linked_list import LinkedList
+
+
+class TestLinkedList(unittest.TestCase):
+
+    def test_attempt_to_add_none_value_raises_type_error(self):
+        linkedlist = LinkedList()
+        try:
+            linkedlist.append(None)
+            self.fail("Should be an exception")
+        except TypeError as e:
+            self.assertEqual("Value is absent!", str(e))
+
+    def test_possible_to_add_negative_value(self):
```

Comment:
Since LinkedList doesn't care about what it stores - it won't matter if it's a negative or positive or zero. I'd write tests like `test_possible_to_add_value_of_any_type` and would append a string, an int, float, etc.